Havit.Blazor.Components.Web.Bootstrap/Chips/HxChipList.razor.cs

```diff
@@ -73,4 +82,14 @@ private async Task HandleRemoveClick(ChipItem chipItemToRemove)
 	{
 		await InvokeOnChipRemoveClickAsync(chipItemToRemove);
 	}
+
+	/// <summary>
+	/// Triggers the <see cref="OnResetClick"/> event. Allows interception of the event in derived components.
+	/// </summary>
+	protected virtual Task InvokeOnResetClickAsync() => OnResetClick.InvokeAsync();
+
+	private async Task HandleResetClick()
+	{
+		await InvokeOnResetClickAsync();
+	}
```

Comment:
You can use `InvokeOnResetClickAsync` from the markup directly, probably no need to have this pass-through method.